recipes/bqplot/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "0.6.1" %}
+
+package:
+    name: bqplot
+    version: {{ version }}
+
+source:
+    fn: bqplot-{{ version }}.tar.gz
+    url: https://pypi.io/packages/source/b/bqplot/bqplot-{{ version }}.tar.gz
+    md5: e0ab4e1f478b85c59e774eb58317cd67
+
+build:
+    number: 1
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - numpy
+        - pandas
+        - jupyter
+    run:
+        - python
+        - ipywidgets >=5.1.3
+        - numpy
+        - pandas
+
+test:
+    imports:
+        - bqplot
+
+about:
+    home: https://github.com/bloomberg/bqplot
+    license: Apache License 2.0
+    summary: Plotting library for Jupyter
+
+extra:
+  recipe-maintainers:
+    - SylvainCorlay
+    - jasongrout
```

Comment:
I'm guessing that you guys talked about this, but figured I should follow up.

Are you ok with being a maintainer of `bqplot`, @jasongrout?